### PR TITLE
manually add twitter-fetcher w/ git auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
@Amomo @Piicksarn could u help me check this PR for #9103? thank you!
**our minify function doesn't support partly minifing files, so I didn't minify un-minfied files.
Or do you have any idea?
I use git auto-update to add old versions before v16.0.2** 

**git repo url:** https://github.com/jasonmayes/Twitter-Post-Fetcher
**Watch 63
  Star 570
 Fork 144**

**npm package url:** https://www.npmjs.com/package/twitter-fetcher
**24 downloads in the last day
118 downloads in the last week
383 downloads in the last month**

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR 
 * [x] The lib has notable popularity  
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry 
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update) 
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/twitter-fetcher/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "homepage": "http://www.jasonmayes.com/projects/twitterApi/#sthash.PwyUJPV4.wJnsNgKd.dpbs",
+  "license": "MIT",
+  "npmName": "twitter-fetcher",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "css/style*",
+        "js/+(twitterFetcher*|exampleUsage*)"
```
Comment (PeterDaveHello) [inline ajax/libs/twitter-fetcher/package.json]: Why do we need `exampleUsage`?

Comment (maruilian11) [inline ajax/libs/twitter-fetcher/package.json]: https://github.com/jasonmayes/Twitter-Post-Fetcher/blob/master/index.html#L35
like style.css is used in index.html for an example usage, I think exampleUsage.js has the same usage.

Comment (PeterDaveHello) [inline ajax/libs/twitter-fetcher/package.json]: No we don't need it.

**Comment by maruilian11** [comment]
@Amomo @Piicksarn I removed `exampleUsage.js`, could you help me check this PR? thank you!

**Comment by PeterDaveHello** [comment]
Don't you need to minify css file?

**Comment by maruilian11** [comment]
@PeterDaveHello sorry about that.
@amomo @Piicksarn I add minified css files for this PR, could you help me check it, thanks!

**Comment by maruilian11** [comment]
ping @Amomo @Piicksarn

**Comment by PeterDaveHello** [comment]
`style.css` seems to be just example ...

**Comment by maruilian11** [comment]
@Piicksarn I turn to use git auto-update config because git has the latest version, and I add v15.0.2 seen on npm package but not no git repo, I have open an issue to ask about the missing version and told the author we prefer npm auto-update.

**Comment by Piicksarn** [comment]
For the homepage config : Using the link shown on the [lib's document](https://github.com/jasonmayes/Twitter-Post-Fetcher#twitter-post-fetcher) is better.

**Comment by maruilian11** [comment]
@Piicksarn thanks! I have modify `homepage` to match with lib repo readme.md.

**Comment by Piicksarn** [comment]
LGTM

**Pull request merged**